app/services/broadcasts/welcome_notification/generator.rb

```diff
@@ -76,7 +76,7 @@ def send_discuss_and_ask_notification
       end
 
       def send_download_app_notification
-        return if user.created_at > 7.days.ago || received_notification?(download_app_broadcast)
+        return if user.created_at.after?(7.days.ago) || received_notification?(download_app_broadcast)
```

Comment:
https://github.com/forem/forem/pull/16052#discussion_r782413219 the surrounding context (when we call this generator from the worker) is there won't be users older than 7 days and this always returns early without sending a notification.

Reply:
I have a pr to fix that.